sale_product_pack/models/sale_order.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import _, api, models
+from odoo.exceptions import UserError
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def copy(self, default=None):
+        sale_copy = super().copy(default)
+        # we unlink pack lines that should not be copied
+        pack_copied_lines = sale_copy.order_line.filtered(
+            lambda l: l.pack_parent_line_id.order_id == self
+        )
+        pack_copied_lines.unlink()
+        return sale_copy
+
+    @api.onchange("order_line")
+    def check_pack_line_unlink(self):
+        """At least on embeded tree editable view odoo returns a recordset on
+        _origin.order_line only when lines are unlinked and this is exactly
+        what we need
+        """
+        if self._origin.order_line.filtered(
```

Comment:
Now with odoo 13, self._origin.order_line is not the deleted line, but the remaining lines as new lines (NewId). So, It can be a posible solution. ping @pedrobaeza 

```suggestion
        origin_line_ids = self._origin.order_line.ids
        line_ids = self.order_line.ids
        removed_line_ids = list(set(origin_line_ids) - set(line_ids))
        removed_line = self.env['sale.order.line'].browse(removed_line_ids)
        if removed_line.filtered(
```